### PR TITLE
Whitelist index.js (which targets NodeJS) for startsWith and console.

### PR DESCRIPTION
**Pull request by powdercloud**

### CODE REVIEW

File: build-system/tasks/presubmit-checks.js [hunk 1]
```diff
@@ -58,6 +58,7 @@ var forbiddenTerms = {
       'validator/validator.js',
       'validator/parse-css.js',
       'validator/validator-in-browser.js',
+      'validator/index.js'  // NodeJs only.
```
Comment (erwinmombay) [inline build-system/tasks/presubmit-checks.js]: add trailing ,

File: build-system/tasks/presubmit-checks.js [hunk 2]
```diff
@@ -262,7 +263,8 @@ var forbiddenTerms = {
     message: es6polyfill,
     whitelist: [
       'validator/tokenize-css.js',
-      'validator/validator.js'
+      'validator/validator.js',
+      'validator/index.js'  // NodeJs only.
```
Comment (erwinmombay) [inline build-system/tasks/presubmit-checks.js]: add trailing ,

**Comment by erwinmombay** [comment]
@powdercloud LGTM, with minor nit

**Comment by powdercloud** [comment]
@erwinmombay Thank you. This made it easy to sort so I did this as well. will merge once Travis is happy.

**Pull request merged**